tf2/include/tf2/convert.h

```diff
@@ -112,12 +109,12 @@ template<typename A, typename B>
  * \param a an object to convert from
  * \param b the object to convert to
  */
-template <class A, class B>
-  void convert(const A& a, B& b)
-  {
-    //printf("In double type convert\n");
-    impl::Converter<ros::message_traits::IsMessage<A>::value, ros::message_traits::IsMessage<B>::value>::convert(a, b);
-  }
+/* template <class A, class B> */
+/*   void convert(const A& a, B& b) */
+/*   { */
+/*     //printf("In double type convert\n"); */
+/*     impl::Converter<ros::message_traits::IsMessage<A>::value, ros::message_traits::IsMessage<B>::value>::convert(a, b); */
+/*   } */
```

Comment:
Nit: please use either block comments for the entire block or `//` for single line comments